tableauserverclient/server/endpoint/workbooks_endpoint.py

```diff
@@ -87,13 +87,19 @@ def update_conn(self, workbook_item, connection_item):
     # Download workbook contents with option of passing in filepath
     @api(version="2.0")
     @parameter_added_in(no_extract='2.5')
-    def download(self, workbook_id, filepath=None, no_extract=False):
+    @parameter_added_in(include_extract='2.5')
+    def download(self, workbook_id, filepath=None, no_extract=None, include_extract=True):
```

Comment:
For some reason I want to move `include_extract` to before `no_extract` so we just trim the end param, but this isn't mandatory, and others may disagree.